explorer/app/viewModelBuilders/anvil-catalog/common/viewModelBuilders.ts

```diff
@@ -140,20 +148,89 @@ export const buildStudyDesigns = (
+    values: anvilCatalogEntity.studyDesign,
   };
 };
 
 /**
- * Build props for terra workspace count cell component from the given AnVIL study.
- * @param anVILCatalogStudy - AnVIL catalog study.
- * @returns Model to be used as props for the terra workspace count cell.
+ * Build props for study design cell component from the given AnVIL entity.
+ * @param anVILCatalogConsortium - AnVIL catalog consortium.
+ * @returns Model to be used as props for the study design cell.
  */
-export const buildTerraWorkspaceCount = (
-  anVILCatalogStudy: AnVILCatalogStudy
-): React.ComponentProps<typeof C.Cell> => {
+export const buildStudyNames = (
+  anVILCatalogConsortium: AnVILCatalogConsortium
+): React.ComponentProps<typeof C.NTagCell> => {
+  return {
+    label: getPluralizedMetadataLabel(METADATA_KEY.STUDY),
+    values: anVILCatalogConsortium.studyName,
+  };
+};
+
+/**
+ * Build props for Details component from the given datasets index or detaset detail response.
+ * TODO revisit - separate from entity builder, generalize modeling/component?, revisit transformer
+ * @param response - Response model return from datasets or dataset API endpoints.
+ * @returns model to be used as props for the Description component.
+ */
+export const buildStudyDetails = (
+  response: AnVILCatalogStudy
+): React.ComponentProps<typeof C.Details> => {
+  const consortium = response.consortium;
+  const phsID = response.dbGapId;
+  const details = new Map<Key, Value>();
+  details.set("Consortium", consortium);
+  details.set("PHSID", phsID);
+
+  return {
+    keyValuePairs: details,
+    title: "",
+  };
+};
+
+/**
+ * Build props for Hero component from the given study response.
+ * TODO revisit - separate from entity builder, generalize modeling?, revisit transformer
+ * @param response - Response model return from datasets API.
+ * @param exploreState - global search state
+ * @returns model to be used as props for the BackPageHero component.
+ */
+export const buildStudyHero = (
+  response: AnVILCatalogStudy,
+  exploreState: ExploreState
+): React.ComponentProps<typeof C.BackPageHero> => {
+  const entityConfig = getEntityConfig(exploreState.tabValue);
+  const firstCrumb = {
+    path: "/" + entityConfig.route,
+    text: entityConfig.label,
+  };
+
+  const studyName = response.studyName;
+  const breadcrumbs = [firstCrumb];
+  if (studyName) {
+    breadcrumbs.push({ path: "", text: studyName });
+  }
+
+  return {
+    breadcrumbs: breadcrumbs,
+    title: response.studyName,
+  };
+};
+
+/**
+ * Build props for study name cell component from the given AnVIL entity.
+ * @param workspaceOrStudy - AnVIL catalog workspace.
+ * @returns Model to be used as props for the study name cell.
+ */
+export const buildStudyName = (
+  workspaceOrStudy: Exclude<AnVILCatalogEntity, AnVILCatalogConsortium>
+): React.ComponentProps<typeof C.Links> => {
   return {
-    value: getTerraWorkspaceCount(anVILCatalogStudy),
+    links: [
```

Comment:
Change the return to suit the `Link` component see comments for `explorer/site-config/anvil-catalog/dev/index/studiesEntityConfig.ts` line `43`.